src/python/zquantum/core/circuit/conversions/pyquil_conversion_test.py

```diff
@@ -376,6 +453,97 @@ def test_dagger_of_orquestra_gate_is_taken_when_converting_gate_with_dagger_modi
         )
 
 
+class TestCustomGateFactoryFromPyquilDefgate:
+    @pytest.mark.parametrize(
+        "name, matrix, qubits",
+        [
+            (
+                "U",
+                [[0.5 + 0.5j, 0.5 - 0.5j], [0.5 - 0.5j, 0.5 + 0.5j]],
+                (3,)
+            ),
+            (
+                "V",
+                [[1, 0, 0, 0], [0, 1, 0, 0], [0, 0, 0, -1j], [0, 0, -1j, 0]],
+                (1, 4)
+            )
+        ]
+    )
+    def test_factory_created_form_nonparametrized_pyquil_defgate_produces_correct_gates(
```

Comment:
`form` -> `from`?

Reply:
Thanks for catching this. Fixed in 8ab2891